remind to update kafka retention policy in kafka disk resize runbook (#2834)
PostHog/posthog.com: contents/docs/self-host/runbook/kafka/resize-disk.md

1. Delete the `StatefulSet` definition but leave its `pod`s online (this is to avoid an impact on the ingestion pipeline availability): `kubectl -n posthog delete sts --cascade=orphan posthog-posthog-kafka`

> 1. In your Helm chart configuration, update the `kafka.persistence` value in `value.yaml` to the target size (20G in this example). Remember to also update the retention policy accordingly, more info [here](/docs/self-host/deploy/troubleshooting#why-did-we-run-into-this-problem-and-how-to-avoid-it-in-the-future)

Author: ```suggestion
1. In your Helm chart configuration, update the `kafka.persistence` value in `value.yaml` to the target size (20G in this example). You might want to update the retention policy too, more info [here](/docs/self-host/deploy/troubleshooting#why-did-we-run-into-this-problem-and-how-to-avoid-it-in-the-future)
```
